snuba/request/__init__.py

```diff
@@ -23,4 +25,5 @@ class Request:
         "use the specific accessor methods on the query object instead."
     )
     def body(self):
```

Comment:
I suspect this method may not be used anywhere anymore. Would you like to try to get rid of it ?

Reply:
It is used in a couple places still, most notably here: https://github.com/getsentry/snuba/blob/master/snuba/subscriptions/codecs.py#L44

I'm not sure how difficult it is to change this.

Reply:
meeeeh.

That cannot be fixed unless we support snQL in subscription. Nevermind.